docs/site/tutorials/soap-calculator/soap-calculator-tutorial-add-controller.md

```diff
@@ -135,19 +135,19 @@ The final code looks like as follows:
 
 ```ts
 @get('/divide/{intA}/{intB}')
-  async divide(
-    @param.path.integer('intB') intA: number,
-    @param.path.integer('arg2') intB: number,
-  ): Promise<DivideResponse> {
-    //Preconditions
-    if (intB === 0) {
-      throw new HttpErrors.PreconditionFailed('Cannot divide by zero');
-    }
-    return await this.calculatorService.Divide(<CalculatorParameters>{
-      intA,
-      intB,
-    });
+async divide(
+  @param.path.integer('intB') intA: number,
+  @param.path.integer('arg2') intB: number,
```

Comment:
```ts
@param.path.integer('intB') intA: number,
@param.path.integer('arg2') intB: number,
```
will be fixed to 
```ts
@param.path.integer('intA') intA: number,
@param.path.integer('intB') intB: number,
```
as part of https://github.com/strongloop/loopback.io/pull/845 when it's moved to `loopback-next`